app/src/main/java/com/kickstarter/viewmodels/SearchViewModel.java

```diff
@@ -96,6 +107,30 @@ public ViewModel(final @NonNull Environment environment) {
       final Observable<String> query = params
         .map(DiscoveryParams::term);
 
+      final Observable<List<Project>> projects = Observable.merge(
+        this.popularProjects.asObservable(),
```

Comment:
This was unnecessary and I removed it. Thanks for finding that issue. Subjects should be naturally observable.